Makefile.js

```diff
@@ -882,6 +882,11 @@ target.checkGitCommit = function() {
 
     // Only check non-release messages
     if (!semver.valid(commitMsgs[0])) {
+        if (commitMsgs[0].slice(0, commitMsgs[0].indexOf("\n")).length > 72) {
```

Comment:
That's not what I saw in `git log`... Under what circumstances does git
automatically add line breaks? Also, if what you say is true, is this even
possible to fix? Also also, couldn't we prove the line break insertion by
having "(fixes" straddle the 72-character mark? If line breaks are in fact
inserted automatically, then checkGitCommit should fail due to the issue
regex not matching.
On Sep 9, 2015 6:48 PM, "Ilya Volodin" <notifications@github.com> wrote:

> In Makefile.js
> <https://github.com/eslint/eslint/pull/3713#discussion_r39111296>:
>
> > @@ -882,6 +882,11 @@ target.checkGitCommit = function() {
> >
> >      // Only check non-release messages
> >      if (!semver.valid(commitMsgs[0])) {
> > +        if (commitMsgs[0].slice(0, commitMsgs[0].indexOf("\n")).length > 72) {
>
> If I'm not mistaken git will automatically add line break at 72
> characters. If I'm correct this will always pass.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eslint/eslint/pull/3713/files#r39111296>.
>